frontend/src/scenes/persons/personsLogic.tsx

```diff
@@ -495,7 +499,7 @@ export const personsLogic = kea<personsLogicType>([
                     actions.navigateToTab(activeTab as PersonsTabType)
                 }
 
-                if (!activeTab) {
+                if (!activeTab && values.activeTab === values.defaultTab) {
```

Comment:
this took forever to find!

it didn't matter what i did this logic was silently replacing parts of the URL

PR that did it mentioned stopping a perf issue, i'm guessing we were setting the URL when we didn't need to, so don't